docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md

```diff
@@ -1,17 +1,18 @@
 ---
 id: troubleshooting
 title: Troubleshooting OpenEBS - Overview
+slug: /troubleshooting/
```

Comment:
why the slug?

Reply:
for converting this `https://gallant-pasteur-5e438c.netlify.app/docs/troubleshooting/troubleshooting` to `https://gallant-pasteur-5e438c.netlify.app/docs/troubleshooting/`

Reply:
oh ok! understood